src/applications/static-pages/BTSSS-login/selectors.js

```diff
@@ -1,14 +1,8 @@
-import { createSelector } from 'reselect';
 import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
 import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 
-const selectFeatureToggles = createSelector(
-  state => ({
-    BTSSSLoginWidget: toggleValues(state)[FEATURE_FLAG_NAMES.BTSSSLoginWidget],
-  }),
-  toggles => toggles,
-);
```

Comment:
Just a note affirming this removal: 

The use of `createSelector` here would likely not result in any performance improvements. Understanding why requires a little longer discussion around `reselect`, but it boils down to the idea that this sort of memoization is only useful when there is more complex calculations going on than just a simple accessing of a state property.